proxy/src/serverless/sql_over_http.rs

```diff
@@ -424,8 +424,8 @@ pub enum SqlOverHttpCancel {
 impl ReportableError for SqlOverHttpCancel {
     fn get_error_kind(&self) -> ErrorKind {
         match self {
-            SqlOverHttpCancel::Postgres => ErrorKind::RateLimit,
-            SqlOverHttpCancel::Connect => ErrorKind::ServiceRateLimit,
+            SqlOverHttpCancel::Postgres => ErrorKind::ClientDisconnect,
```

Comment:
Is this always `clientdisconnect`?

Reply:
It is now. The only time a db request is cancelled is if the HTTP connection is closed, since we removed the prior timeout code